### ISSUE TITLE
Volume Control Option

### ISSUE THREAD

**Issue by Sickjuicy** [ytmdesktop/ytmdesktop#1198]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Feature

A option so I can use the scroll wheel on the volume silder on a pre configert percentage like if I want to have it set to 1% per scroll click or 5% so the volume steps are than 5% 10% 15% or at 1%, 1% 2% 3% 4%.

**Comment by Alipoodle** [comment]
Have you tried using the Ratio Volume instead 😄 (dB audio in YTMD v1)
This allows for a wider range of lower Volumes compared to native YTM so you can enjoy some quite sounds in the background.

**Comment by Sickjuicy** [comment]
Yes I use it, but even there I want to have more freedom xD Its okey if you guys dont implement it but would be cool even if it is in the config file.
